src/main/resources/application.yaml

```diff
@@ -26,6 +26,8 @@ auction:
 amp:
   default-timeout-ms: 900
   timeout-adjustment-ms: 30
+setuid:
+  default-timeout-ms: 2000
```

Comment:
Is it used anywhere?

Reply:
it was deleted from everywhere but forgot in default app config. Will fix.